### PR TITLE
Wisenet Wave integration (Script)

### PR DESCRIPTION
**Pull request by ttayson**
Wisenet Wave integration

This integration is similar to the current NX integration, but it now adds bookmarks instead of notifications. This way, license plate information is stored in the software and can be accessed later, allowing the user to instantly access recorded video.

It utilizes the recommended session cookie from the latest API and does not require the use of Digest user (not recommended by the API). Session renewal occurs every 24 hours.

This integration works for both NX/Wisenet and DW.

### CODE REVIEW

**Review by marcbelmont — COMMENTED** [review]
- [ ] Update `webhooks/README.md` to include information about the new integration.

File: webhooks/Webhook_Wisenet/main.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        response.raise_for_status()
+        return jsonify({"success": "request sent"}), response.status_code
+    except (json.JSONDecodeError, requests.exceptions.RequestException) as err:
+        logging.error(f"Error processing the request: {err}")
+        return jsonify({"error": "Failed to process the request"}), 500
+
+
+if __name__ == "__main__":
+
+    from waitress import serve
```
Comment (marcbelmont) [inline webhooks/Webhook_Wisenet/main.py:104]: What are the advantages of using Waitress instead of serving directly with Flask?

Comment (ttayson) [reply webhooks/Webhook_Wisenet/main.py:104]: Just to remove the warning messages that Flask shows

2023-10-20 22:45:13,777 - INFO: WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.

do you have any simpler options?

Comment (marcbelmont) [reply webhooks/Webhook_Wisenet/main.py:104]: you can try https://stackoverflow.com/questions/14888799/disable-console-messages-in-flask-server

Comment (ttayson) [reply webhooks/Webhook_Wisenet/main.py:104]: @marcbelmont changing log level didn't work

**Review by AashishDhakal — APPROVED** [review]

**Comment by marcbelmont** [comment]
@ttayson I don't see the changes to https://github.com/parkpow/deep-license-plate-recognition/blob/master/webhooks/README.md

**Comment by ttayson** [comment]
> @ttayson I don't see the changes to https://github.com/parkpow/deep-license-plate-recognition/blob/master/webhooks/README.md

I @marcbelmont, sorry for delay. I have edited the Readme for the subproject itself, and I have added a brief explanation of the subproject. Can we leave it like this?

If yes, I will also do the same with Webhook_OpenEye to avoid code repetition

**Review by marcbelmont — APPROVED** [review]

**Comment by marcbelmont** [comment]
@ttayson, yes, it's good that way.

**Pull request merged**